### PR TITLE
Update to include thirdweb-dev & thridwed-examples

### PR DESCRIPTION
**Pull request by BankkRoll**
Update response to include thirdweb-dev & thridwed-examples to show all "Top Community Contributors"

### CODE REVIEW

**Comment by jarrodwatts** [comment]
Great idea @BankkRoll!

This looks good to me  - will get a second opinion from one of our engineers before merging

**Comment by BankkRoll** [comment]
> Great idea @BankkRoll!
> 
> This looks good to me - will get a second opinion from one of our engineers before merging


Thanks @jarrodwatts!
I took note of the filter in the repo results. Made sure we didn't need to filter some from examples :)

File: pages/open-source.tsx
Comment (jnsdls) [inline pages/open-source.tsx:330]: could do this simultaneously instead of one at a time (use promise all, merge the results and then process them all together)

File: pages/open-source.tsx
Comment (jnsdls) [inline pages/open-source.tsx:344]: same thing here this should be done in parallel instead of one at a time (using promise.all)

**Review by jnsdls — CHANGES_REQUESTED** [review]
fix merge conflicts and parallelize requests then ship

**Comment by jarrodwatts** [comment]
Hey @BankkRoll - feel free to ping me on Discord if you have any questions on these!

**Comment by BankkRoll** [comment]
> Hey @BankkRoll - feel free to ping me on Discord if you have any questions on these!

Hey sounds good just seeing this, will get a look and update this

**Comment by BankkRoll** [comment]
@jarrodwatts idk what i did lol im tired and closed it again... ill go back and update it and sumbit the pr again or reopen this tomorrow to tired tonight, i accidently closed it trying to sync and merge the new updates.

**Comment by BankkRoll** [comment]
Hey @jarrodwatts not sure if that was good enough but best way I figured out how to fix whatever I did. and swapped the fetching to use promise all updated pr #1200 , also mentioned this PR

**Pull request closed without merging**
